packages/select/src/common/listItemsProps.ts

```diff
@@ -106,6 +106,13 @@ export interface IListItemsProps<T> extends IProps {
      */
     resetOnSelect?: boolean;
 
+    /**
+     * Whether the active item should be reset to first value when a query is
+     * entered. The first item will be made active.
+     * @default false
+     */
+    resetActiveItemOnQuery?: boolean;
```

Comment:
ok so I see why you called it `resetActiveItemOnQuery` instead of `resetOnQuery`, but i kinda feel like the mere fact that it has `query` in the name suggests it won't reset _the query_ on query. we can add a note in the prop docs to clarify. thoughts?

Reply:
Yeah, I don't love the name either but felt it was clearer than `resetOnQuery`. I'm open to a better name and am happy to expand the docs a bit.

Reply:
let's go with `resetOnQuery` and see how it feels. i like the consistency.

i'm definitely inclined towards `true` by default -- it's the behavior i've been trying to implement all along. question is if it noticeably breaks common use cases?

Reply:
Ok, I think that prop name is a bit unclear, but will push up a build with that change & defaulting to `true`. Playing with the demo a bit feels good -- I don't think that it would break any common use cases.